templates/feature.html

```diff
@@ -82,44 +82,30 @@
           {% else %}
             Enabled by default
           {% endif %}
-        {% endif %} in <b>desktop Chrome {{ feature.meta.milestone_str }}</b>
+        {% endif %}
+        {% if feature.bug_url %}
+        (<a href="{{ feature.bug_url }}">launch bug</a>)
+        {% endif %}
+        in: </br>
```

Comment:
Do you mean just `<br>`?  br is an [empty element](https://developer.mozilla.org/en/docs/Web/HTML/Element/br).